src/main/resources/db/migration/V42__opprett_tabell_tilbakekreving_snapshot.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE tilbakekreving_snapshot (
+    id UUID PRIMARY KEY,
+    snapshot JSONB NOT NULL
+);
```

Comment:
Slenger du på en trailing new line?
```suggestion
);

```